packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -133,6 +176,19 @@ export class Tabs extends React.Component<TabsProps & OUIAProps, TabsState> {
     }
   };
 
+  // Format the inset prop string by capitalizing the first letter.  If the string prop is '2xl' append '_' to beginning of the string so we can key the correct modifier.
+  formatInsetString = (s: string) => (s === '2xl' ? '_2xl' : capitalize(s));
+
+  componentDidMount() {
+    window.addEventListener('resize', this.handleScrollButtons, false);
```

Comment:
Should we only listen to resize if `!isVertical`? Looks like it needs that condition to add the modifier